lib/recipe_box/cuisine.ex

```diff
@@ -0,0 +1,46 @@
+defmodule RecipeBox.Cuisine do
+  @moduledoc """
+  This is the Cuisine context
+  """
+
+  use Ecto.Schema
+  import Ecto.Changeset
+  
+  alias RecipeBox.Cuisine
+
+  @doc "RecipeBox.Repo is the application's Ecto repository"
+  alias RecipeBox.Repo
+  
+  @doc """
+  This specifies the database table and corresponding Elixir struct at the same time.
+  Generate a migration file by running mix ecto.gen.migration create_cuisines.
+  """
+  schema "cuisines" do
+    field :name, :string
+
+    timestamps()
+  end
+  
+  @doc """
+  Changesets allow Ecto to manage record changes.
+  The cast function allows only the specified fields into the DB and nothing else.
+  """
+  def changeset(cuisine, attrs) do
+    cuisine
+    |> cast(attrs, [:name])
+    |> validate_required([:name])
+    |> downcase_value
```

Comment:
I like how you're ensuring names don't get duplicated here, but this has the side-effect of making all cuisine names always lowercase. That may be OK, but it's something to think about, since a user might expect to have the capitalization they typed in be preserved. Not necessarily a bug, but something to think about!

Reply:
Thanks! I definitely have this on my mind. I was thinking of it as something to tackle when I work on the styling. Using CSS to make it Title Case or something. I wonder now though, is it better to preserve input as the user types it in, or format it for styling purposes?

Reply:
I think more often than not, it's better to preserve things as the user input them. There are definitely times when it makes sense to stray from that, but I think it's a good rule of thumb. As a bad, made-up example, it wouldn't work to CSS title case a cuisine called `My iPad Recipes`.